.github/workflows/gradle.yml

```diff
@@ -3,21 +3,20 @@ name: Java & Go CI
 
 on:
   push:
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths-ignore:
+      - 'docs/**'
+      - 'github-actions/**'
+      - 'sechub-doc/**'
+      - 'sechub-website/**'
     # We ignore everything where tag starts with v* - this is done by release build!
     tags-ignore:
       - v*
   pull_request:
-    # This workflow will run for all pull requests that target following branches
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths-ignore:
+      - 'docs/**'
+      - 'github-actions/**'
```

Comment:
this is problematic
since you made changes to `gradle.yml` this PR should also execute the Java Build Pipeline, but it doesn't
This is because `github-actions/**` ignores changes to all workflow files

Reply:
The workflow files are under `.github`.
Unfortunately this path (or everything starting with a `.` seems to be ignored by default. :disappointed: 
Changed the filter so the gradle.yml is included...